src/aws-lambda/cart-event-consumer/stage.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
this file looks very generic, why is it in the "cart-event-consumer" folder = can it be in aws-lambda for more reuse?

Reply:
I'm just copying what's there already, the `stage.sh` file in the root directory relies on this folder structure and those two scripts being a lambda dir so I needn't want to start hacking at that to make it more generic. I think changing this should probably done in `aws-samples/retail-demo-store` anyway and ported to here. I will raise an issue there.

Reply:
ok raise the issue in the upstream .
but for our work for now, it's fine